client/cluster-client/src/main/kotlin/uk/ac/ebi/biostd/client/cluster/api/LsfClusterClient.kt

```diff
@@ -90,7 +90,6 @@ class LsfClusterClient(
     companion object {
         const val DONE_STATUS = "DONE"
         const val EXIT_STATUS = "EXIT"
-        const val PEND_STATUS = "PEND"
```

Comment:
why we have remove this one? (I remember seeing as a valid state)

Reply:
It was never used. Maybe we need to do something with it

Reply:
yes is not used in a particular test but it is used when obtained job status using API